### DISCUSSION TITLE
How to add directory name to output?

### DISCUSSION THREAD

**Discussion by snwefly** [mifi/lossless-cut#1732] [Q&A]
I like to lossless-cut to export all clips to currentdirectory/segment/  to keep structure clean.

I tired adding segment/ to output template and file name preview is fine but getting an error when i try to export the clip?

![image](https://github.com/mifi/lossless-cut/assets/88805679/ac7c3ddb-a41e-4257-8264-14c15536dccd)

**Comment by mifi** [comment]
does it work if you use windows backslash instead? `\`

**Reply by snwefly** [reply]
It won't let me add \ via ui

![image](https://github.com/mifi/lossless-cut/assets/88805679/93227488-1672-48e2-b132-87e6718f22ed)

I added \ via config.json and program completely ignored the config and used default setting for ouput.

**Reply by mifi** [reply]
I think you need to turn off "Sanitize"

**Reply by snwefly** [reply]
didn't work 
![image](https://github.com/mifi/lossless-cut/assets/88805679/2414f41a-bdbf-46eb-850a-e49aad6d960b)

**Reply by mifi** [reply]
Does it work if you use forward slash `/` with `No sanitize`?

**Reply by mifi** [reply]
if not, could you check if there's an error in the developer tools?
